int32 xmin
int32 ymin
int32 xmax
int32 ymax
float32 sudut
int32 colorIdx